examples/examples.pro

```diff
@@ -0,0 +1,32 @@
+TEMPLATE = subdirs
+SUBDIRS += example01 \
+    example02 \
+    example03 \
+    example04 \
+    example05 \
+    example06 \
+    example07 \
+    example08 \
+    example09 \
+    example10 \
+    example11 \
+    example12 \
+    example13
```

Comment:
You can directly enumerate the .pro files here

Reply:
Thanks, I've done so.